docs/tutorial/audio2text/audio2text.md

```diff
@@ -0,0 +1,77 @@
+# Creating an Audio to Text App with Jina and DocArray V2
+
+This is how you can build an Audio to Text app using both Jina and DocarrayV2
+
+We will use: 
+
+* DocarrayV2: Helps us to load and preprocess multimodal data such as image, text and audio in our case
+* Jina: Helps us serve the model quickly and create a client
+
+First let's install requirements
+
+## 💾 Installation
+
+```bash
+pip install -r requirments.txt
```

Comment:
lets just put pip install one by one. Because from the documentation user cannot see the requirement file

```
pip install transformer
pip install whisper
...
```